packages/blade/src/components/Input/SelectInput/SelectInput.tsx

```diff
@@ -85,7 +83,7 @@ const _SelectInput = (
         {...baseInputProps}
         as="button"
         componentName={MetaConstants.SelectInput}
-        ref={triggererRef as React.MutableRefObject<HTMLInputElement>}
+        ref={!isReactNative() ? (triggererRef as React.MutableRefObject<HTMLInputElement>) : null}
```

Comment:
React Native Testing library doesn't support refs https://github.com/callstack/react-native-testing-library/issues/1069#issuecomment-1221105858

There are some workarounds but we're anyway not using this ref in React native so removed it from here